app/src/main/java/com/cornellappdev/volume/ui/components/general/PublicationComponents.kt

```diff
@@ -70,6 +70,7 @@ fun CreateHorizontalPublicationRow(
                     fontWeight = FontWeight.Medium,
```

Comment:
Please add an onPublicationClick to the CreateHorizontalPublicationRowFollowing, the publications from More Publications are not clickable